roles/timer/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Fail when timer_name is empty
+  fail: msg="timer_name is empty"
+  when: not timer_name
+
+- name: Upload service and timer
+  template:
+    src: "job.{{ item }}"
+    dest: /lib/systemd/system/{{ timer_name }}.{{ item }}
+  with_items:
+  - service
+  - timer
+  register: template_result
+
+- name: Reload systemd
+  command: systemctl daemon-reload
+  when: template_result.changed
+
+- name: Enable and start timer
```

Comment:
Should we always enable and start the Timer?
Maybe a deployment of generic timers would be nice - ala:
`update-fastd-blacklist@.service` where the variable `%I` could be the `{{ site_code }}`